specs/informasjons_og_datamodell.md

```diff
@@ -708,6 +716,7 @@ Lekkasje av PII kan oppstå mellom flere parter i verdikjeden:
 * mellom informasjonstjeneste og datagrensesnitt
 
 For å sikre oss mot potensiell lekkasje av PII bør det vurderes å innføre tiltak for å ivareta konfidensialiteten.
+> Kommentar TRE: JWT og SAML-tokens transporteres i en HTTPS request i hhv. HTTP Authorization Header og HTTP Body som Header i SOAP-melding. Sistnevnte er da like godt/dårlig beskyttet som selve payload i SOAP-meldingen. For førstnevnte, så er det jo en generell utfording av HTTPS termineres i transportlaget. Det gjelder jo både HTTP Headers og Body. Må man man ha ende-til-ende-kryptering, så er vel VPN eneste mulighet dersom man ikke implementerer dette i applikasjonslaget (JWE) og kryptering av payload i HTTP Body.
```

Comment:
Kjempebra innspill, og veldig i tråd med vår forståelse i NHN. 
Det vil bli gjennomført en DPIA, hvor vi i etterkant må se på sikkerhetstiltak. Jeg tror vi må gjøre en risikovurdering før vi bestemmer oss for sikkerhetstiltak, f.eks. må vi forstå hvor stor risiko det faktisk er knyttet til terminering av TLS før vi innfører kryptering av tokens. Personlig mener jeg at man har andre utfordringer dersom TLS termineres ett sted hvor man ikke har kontroll på logging.

Fint om du kan fjerne kommentaren, og opprette en ny diskusjon eller et issue som er knyttet til spesifikasjonen om bruk av OIDC og OAuth. Jeg mener at vi må peke på eventuelle personvernsutfordringer i informasjonsmodellen, men skille diskusjonene om risiko og tiltak fra selve spesifikasjonen som et eget tema.